### DISCUSSION TITLE
The “align” light is not on

### DISCUSSION THREAD

**Discussion by zrh114311** [flybywiresim/aircraft#2718] [Help] [answered]
when I turn the ADIRS into “NAV”，the “align” light is not on，only the “fault” light flashed and then went outimmediatly，why？The estimated time is still displayed on the ECAM.

**Comment by hotshotp** [comment]
there is no align light, its only on bat and fault. on bat we cant edit right now but fault is displayed correctly

**Reply by zrh114311** [reply]
![Screenshot_20201226_144220_com huawei himovie](https://user-images.githubusercontent.com/51064462/103146786-08489c00-4789-11eb-94d5-b0589f13ab99.jpg)
Emm，but I saw Fly by wire’s video on YouTube，the align light is on，So how to explain this?

**Reply by hotshotp** [reply]
hmm, i see. disregard my comment then, you are right

**Comment by Inaxair** [comment]
Do you use any other mods?

**Comment by hotshotp** [comment, marked as answer]
just read up on this, it was removed because it was unrealistic
